src/things/person.cpp

```diff
@@ -0,0 +1,63 @@
+/*
+ * Authors : Robertas Norkus
+ * Organisation: HYPED
+ * Date: 2. November 2019
+ * Description:
```

Comment:
Add description.

Reply:
That was done unintentionally. Thanks for pointing that out!